meta-lmp-base/classes/fip-utils.bbclass

```diff
@@ -0,0 +1,391 @@
+DEPENDS += "tf-a-tools-fio-native"
```

Comment:
Can you separate the inclusion of fip-utils.bbclass into 2 commits? One importing from meta-st (saying the reference hash used) and another one with our changes on top?